### DISCUSSION TITLE
Playback of DASH/HLS

### DISCUSSION THREAD

**Discussion by iannock** [moneytoo/Player#300] [Q&A]
In the brief for the app, it says it supports DASH/HLS playback - but there is no URL  #entry capability - how do you do this?

**Comment by moneytoo** [comment]
Tap the link and open it in Just Player. Alternatively, share it (the url) to Just Player which also starts the playback.

**Reply by R0bur** [reply]
Hello!

I tried your app (version 0.136) for playing HLS streams. Sometime it works fine but other times I encounter 'loops' (it plays one part again and again) or 'froze'. I think this depends on the speed of the stream and network condition. MPV player doesn't have such troubles with the network streams playing. Would you add these two features to your application:

1) an ability to interactive switch the HLS stream speed (480, 720, ...);
2) a setting to specify the network buffer size?

Unfortunately I can't provide the stream URL because it is dynamic and expires quickly.
